src/v/kafka/server/handlers/fetch.h

```diff
@@ -205,10 +202,10 @@ struct fetch_config {
 };
 
 struct ntp_fetch_config {
-    ntp_fetch_config(model::ktp ktp, fetch_config cfg)
```

Comment:
the fetch_config is quite big and moving it is not free, so we avoid some moves here and elsewhere (e.g., using emplace_back)